adele-payment-manager/src/test/java/com/microservicesteam/adele/payment/PaymentManagerTest.java

```diff
@@ -1,34 +1,72 @@
 package com.microservicesteam.adele.payment;
 
-import static com.microservicesteam.adele.payment.PaymentStatus.CREATED;
-import static java.math.BigDecimal.TEN;
 import static org.assertj.core.api.Assertions.assertThat;
-
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.util.Currency;
+import static org.mockito.Mockito.when;
 
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.microservicesteam.adele.payment.paypal.PaymentRequestMapper;
+import com.microservicesteam.adele.payment.paypal.PaymentResponseMapper;
+import com.microservicesteam.adele.payment.paypal.PaypalProxy;
+import com.paypal.api.payments.Payment;
+import com.paypal.base.rest.PayPalRESTException;
 
+@RunWith(SpringRunner.class)
```

Comment:
Maybe it's nitpicking, but I feel that `SpringRunner` is a bit overkill here. Why not use `MockitoJUnitRunner` instead?

Reply:
Totally right